SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2514,6 +2524,13 @@ void USpatialNetDriver::HandleStartupOpQueueing(TArray<SpatialGDK::OpList> InOpL
 
 		if (bIsReadyToStart)
 		{
+			ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(GetWorld()->GetWorldSettings());
+#if WITH_EDITORONLY_DATA
```

Comment:
Just wondering if there is a reason to use WITH_EDITORONLY_DATA rather than WITH_EDITOR?

Reply:
I think theoretically you can build non-editor targets, like the standalone game, but keeping the editor data, for tooling.
For eg, we could think about having a test runner outside the editor.